metadata-ingestion/src/datahub/ingestion/source/nifi.py

```diff
@@ -1002,7 +1027,7 @@ def make_external_url(
             component_id = parent_rpg_id
         return urljoin(
             self.config.site_url,
-            f"/nifi/?processGroupId={parent_group_id}&componentIds={component_id}",
```

Comment:
site_url must always end with /nifi/. This handling is moved into pydantic validator for site_url